optuna_integration/tfkeras.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+from typing import Dict
+from typing import Optional
+import warnings
+
+import optuna
+
+
+with optuna._imports.try_import() as _imports:
```

Comment:
Could you use `optuna_integration._imports` instead?

Reply:
Sorry for the confusion. I forgot to note this on the issue https://github.com/optuna/optuna/issues/4484.

Reply:
Ah, https://github.com/optuna/optuna-integration/pull/16 introduced `optuna_integration._imports` yesterday. I'll rebase the master and apply it to this PR.

Reply:
Fixed in https://github.com/optuna/optuna-integration/pull/21/commits/1dd2ebdb8aa183558b66c0f732381fe501e7c2d7.